index.html

```diff
@@ -0,0 +1,232 @@
+  </head>
+  <body>
+    <!-- I suggest we use the flex displaying to better handle the direction and sizes of boxes -->
+    <header>
+    <nav>
+      <!-- In the navbar, we first have image, the first is the 3 bands and the secon is the logo -->
+      <div class="logo-container">
+        <a href="#subscribe"><img src="./img/humburgerIcon.png" alt="Menu icon" class="menu"/></a>
+        <img src="./img//youtube1.png" alt="youtube logo" class="logo" />
+      </div>
+      <input
+        class="search"
+        type="text"
+        name="search"
+        placeholder="Search"
+      /><span><button class="btn-search">Search</button></span>
+      <div class="btn-section">
+        <img src="./img/videoIcon.png" alt="icons" class="icons" />
+        <img src="./img/menuIcon.png" alt="icons" class="icons" />
+        <img src="./img/notifIcon.png" alt="icons" class="icons" />
+        <img src="./img/icons8-male-user-100.png" alt="avater" class="avatar" />
+      </div>
+    </nav>
+    </header>
+    <main>
+    <div class="pageContent">
+        <div class="content">
+          <div class="VideoContainer">
+            <iframe class="video-play"
+              src="https://www.youtube.com/embed/tgbNymZ7vqY">
+            </iframe>
+            <div class="video-title">
+              <h2>Contributing to Open Source Part I: The Easy Way</h2>
+              <div class="sub-title">
+                <div class="details">
+                  <p>61,194 views•Feb 27, 2014</p>
+                </div>
+                <div class="icons-video">
+                  <div class="icons-with-numbers">
+                    <img src="./img/like.png" alt="icons" class="icons" /><p>486</p>
+                  </div>
+                  <div class="icons-with-numbers">
+                    <img src="./img/dislike.png" alt="icons" class="icons" /><p>12</p>
+                  </div>
+                  <div class="icons-with-numbers">
+                    <img src="./img/share.png" alt="icons" class="icons" /><p>SHARE</p>
+                  </div>
+                  <div class="icons-with-numbers">
+                    <img src="./img/save.png" alt="icons" class="icons" /><p>SAVE</p>
+                  </div>
+                </div>
+              </div>
+              
+            </div>
+          </div>
+          <hr>
+          <div>
+            <div class="channel">
+              <div class="channel-details"">
+                <img src="./img/logo.png" alt="icons" class="logo-channel" />
+                <div>
+                  <p>The oden Project</p>
+                  <p class="channel-subscribers"</p>
+                </div>
+              </div>
+              <div class="subscribe-btn">
+                <img src="./img/subscribe.jpg" alt="Subscribe button" />
+              </div>
+            </div>
+          </div>
+          <div class="iconPart"></div>
+          <div class="description">
+            <!-- <h3>Published on Feb 27, 2014</h3> -->
```

Comment:
Please remove this commented code, before you merge changes to the base branch

Reply:
Noted dear, thank you